roles/template_service_broker/vars/default_images.yml

```diff
@@ -1,4 +1,4 @@
 ---
 __template_service_broker_prefix: "docker.io/openshift/"
-__template_service_broker_version: "latest"
+__template_service_broker_version: "{{ openshift_template_service_broker_image_tag | default('latest') }}"
```

Comment:
This should also be openshift_release, I think. It was probably an oversight that this was set to latest in the first place.

Reply:
Good catch.

This one would be a bit trickier, as Origin TSB image doesn't have `v3.7` tag, instead it has `v3.7.0` - and openshift-enterprise images do have `{{openshift_release}}` tag.

Reply:
Lets use `{{ openshift_image_tag }}` for this